tasks/ubuntu.yml

```diff
@@ -1,6 +1,8 @@
 ---
 - name: Add Zulu.org JDK repository key
-  apt_key: keyserver=keyserver.ubuntu.com id=0x219BD9C9
+  apt_key:
+    keyserver: keyserver.ubuntu.com
+    id: 0x219BD9C9
```

Comment:
This might be an old GPG key, see http://docs.azul.com/zulu/zuludocs/#ZuluUserGuide/PrepareZuluPlatform/AttachAPTRepositoryUbuntuOrDebianSys.htm

To avoid that problem in the future, this should be a variable with a default value of a current GPG key. Should it change in the future, at least that would be overwrite by a playbook, before this role is updated

Reply:
It worked, but in general good point. I'm going to create an 'issue' to fix it during next round of updates.

Reply:
Actually we might need to drop this all together following end of live for Java 8 (no free OracleJDK -> none OpenJDK updates)

Reply:
That particular GPG key *didn't* work for me when I started Vagrant boxes from example folder. That is how I found that issue.

Reply:
fixed.